README.rst

```diff
@@ -39,6 +39,16 @@ License
 ``giotto-ph`` is distributed under the AGPLv3 `license <https://github.com/giotto-ai/giotto-tda/blob/master/LICENSE>`_.
 If you need a different distribution license, please contact the `L2F team`_.
 
+Parallel persistent homology backend
+====================================
+
+Computing persistence barcodes of large datasets and in high homology degrees is challenging even on modern hardware. ``giotto-ph``'s persistent homology backend
+is able to distribute the key stages of the computation (namely, search for apparent pairs and coboundary matrix reduction) across an arbitrary number of available CPU threads.
+The scaling is generally quite favourable as shown in the following figure (for more details, see our paper linked below):
+
+.. image:: https://raw.githubusercontent.com/giotto-ai/giotto-ph/readme_speedup_figure/docs/images/multithreading_speedup.svg
```

Comment:
I would have a preference for centering the image, what do you think ? (https://stackoverflow.com/a/67479088)

Reply:
I tried placing the centering directive, but it doesn't work.  Apparently it is a known issue with GitHub: https://github.com/github/markup/issues/163

Reply:
Oh, I did not know.
Apparently in the last answer from the issue, you could hack this by providing directly the html code.
See https://github.com/github/markup/issues/163#issuecomment-846676443

Reply:
Yep, let me try this approach.

Reply:
I can't make it work!

Reply:
Reverted to standard RST image directives